RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py

```diff
@@ -310,13 +322,50 @@
     TOB5 = dict(clusterChargeCut = dict(refToPSet_ = "SiStripClusterChargeCutNone")),
     TOB6 = dict(clusterChargeCut = dict(refToPSet_ = "SiStripClusterChargeCutNone")),
 )
+_convLayerPairs = cms.EDProducer("SeedingLayersEDProducer",
+                                layerList = cms.vstring('BPix1+BPix2',
+                                                        'BPix2+BPix3',
+                                                        'BPix3+BPix4',
+
+                                                        'BPix1+FPix1_pos',
+                                                        'BPix1+FPix1_neg',
+                                                        'BPix2+FPix1_pos',
+                                                        'BPix2+FPix1_neg',
+                                                        'BPix3+FPix1_pos',
+                                                        'BPix3+FPix1_neg',
+
+                                                        'FPix1_pos+FPix2_pos',
+                                                        'FPix1_neg+FPix2_neg',
+                                                        'FPix2_pos+FPix3_pos',
+                                                        'FPix2_neg+FPix3_neg'
+                                                        ),
+
+                                BPix = cms.PSet(
+                                    hitErrorRZ = cms.double(0.006),
+                                    hitErrorRPhi = cms.double(0.0027),
+                                    TTRHBuilder = cms.string('TTRHBuilderWithoutAngle4PixelPairs'),
+                                    HitProducer = cms.string('siPixelRecHits'),
+                                    useErrorsFromParam = cms.bool(True),
+                                    skipClusters = cms.InputTag('convClusters'),
+                                    ),
+                                FPix = cms.PSet(
+                                    hitErrorRZ = cms.double(0.0036),
+                                    hitErrorRPhi = cms.double(0.0051),
+                                    TTRHBuilder = cms.string('TTRHBuilderWithoutAngle4PixelPairs'),
+                                    HitProducer = cms.string('siPixelRecHits'),
+                                    useErrorsFromParam = cms.bool(True),
+                                    skipClusters = cms.InputTag('convClusters'),
+                                    )
+)
+eras.trackingPhase2PU140.toReplaceWith(convLayerPairs, _convLayerPairs)
```

Comment:
The same effect should be able to be achieved with `toModify()`. With EDProducers etc. `toReplaceWith()` should be necessary only when the C++ class name changes. I'm not strongly suggesting that though, as this approach could be more clear (I'm not sure at the moment). If you decide to stay, either change this to
```
eras.trackingPhase2PU140.toReplaceWith(convLayerPairs, cms.EDProducer("SeedingLayersEDProducer",
  ...
)
```
or rename `_convLayerPairs` to `_convLayerPairs_trackingPhase2PU140` (or similar).